packages/web-app-text-editor/src/App.vue

```diff
@@ -116,8 +116,7 @@ export default {
         .then(
           (response) => {
             serverContent.value = newContent
-            // FIXME: above we need response.headers.ETag, here we need response ETag - feels inconsistent
```

Comment:
Note: I removed this comment, as it is intended, above we do a get request to a file (where the headers should contain the etag), while updating a resource the etag is emitted in the payload, which is correct